test/Microsoft.ML.Predictor.Tests/TestAutoInference.cs

```diff
@@ -58,21 +58,21 @@ public void TestLearn()
                 // REVIEW: Theoretically, it could be the case that a new, very bad learner is introduced and 
                 // we get unlucky and only select it every time, such that this test fails. Not
                 // likely at all, but a non-zero probability. Should be ok, since all current learners are returning d > .80.
-                double d = bestPipeline.RunTrainTestExperiment(datasetTrain, datasetTest, metric, MacroUtils.TrainerKinds.SignatureBinaryClassifierTrainer);
-                env.Check(d > 0.2);
+                //double d = bestPipeline.RunTrainTestExperiment(datasetTrain, datasetTest, metric, MacroUtils.TrainerKinds.SignatureBinaryClassifierTrainer);
+                //env.Check(d > 0.2);
```

Comment:
any reason for commenting this out?

Reply:
Don't know why it was commented out. Uncommenting.